src/Microsoft.ML.Tokenizers/Microsoft.ML.Tokenizers.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$(RepoRoot)eng/pkg/Pack.props" />
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
For this project we probably want to keep netstandard2.0.  We could consider cross-targeting if we want to benefit from net6.0+/net8.0+ APIs.